app/controllers/api/pods_controller.rb

```diff
@@ -113,8 +113,17 @@ def verify_pushes_allowed!
                       'your version of CocoaPods to push this specification.'
             json_error(422, message)
           end
+
+          if Version.new(version[1]) == Version.new("1.3.0") 
```

Comment:
Ah sorry I missed that part. You could grep for a `test_type` key. That would work as well.